react-components/src/components/RevealToolbar/SetFlexibleControlsType.tsx

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement, useEffect, useState } from 'react';
+
+import { SegmentedControl, Tooltip as CogsTooltip, type IconType } from '@cognite/cogs.js';
+import { useReveal } from '../RevealCanvas/ViewerContext';
+import {
+  type CameraManager,
+  FlexibleControlsType,
+  type IFlexibleCameraManager
+} from '@cognite/reveal';
+
+import { useTranslation } from '../i18n/I18n';
+
+type CustomSettingsProps = {
+  includeOrbitInCenter?: boolean;
+};
+
+export function SetOrbitOrFirstPersonControlsType(): ReactElement {
+  return SetFlexibleControlsType({ includeOrbitInCenter: false });
+}
+
+export function SetFlexibleControlsType({
+  includeOrbitInCenter
+}: CustomSettingsProps): ReactElement {
+  const viewer = useReveal();
+  const manager = asFlexibleCameraManager(viewer.cameraManager);
+  const { t: translate } = useTranslation();
+
+  // This value is redunant (react force me to do it)
+  const [selectedValue, setSelectedValue] = useState<FlexibleControlsType>(
+    getDefaultValue(manager)
+  );
```

Comment:
Could we have somewhat more descriptive variable names?

Reply:
mmm. It's only one thing that can be selected here

Reply:
This is now fixed